---
layout: archive
title: "Publications"
permalink: /publications/
author_profile: true
---

You can also find these on <a href="https://scholar.google.ca/citations?user=o00Z6d0AAAAJ&hl=en&oi=ao">Google Scholar</a>.

<small><em>* co-first author, <sup>†</sup> corresponding author</em></small>

**Preprints**

<ol class="publications" reversed start="33">

    <li>Squair, J.W., Gautier, M., Kathe, C., Anderson, M.A., James, N.D., Hutson, T.H., Hudelle, R., Qaiser, T., Matson, K.J.E., Barraud, Q., Levine, A.J., La Manno, G., <u>Skinnider, M.A.</u><sup>†</sup>, Courtine, G.<sup>†</sup> &ldquo;Confronting false discoveries in single-cell differential expression.&rdquo; <em>bioRxiv</em>, doi: <a href="https://www.biorxiv.org/content/10.1101/2021.03.12.435024v1">10.1101/2021.03.12.435024</a>. </li>
    
    <li><u>Skinnider, M.A.</u>, Rogalski, J., Tigchelaar, S., Manouchehri, N., Prudova, A., Jackson, A.M., Nielsen, K., Jeong, J., Chaudhary, S., Shortt, K., Gallagher-Kurtzke, Y., So, K., Fong, A., Gupta, R., Okon, E.B., Rizzuto, M.A., Dong, K., Streijger, F., Belanger, L., Ritchie, L., Tsang, A., Christie, S., Mac-Thiong, J.-M., Bailey, C., Ailon, T., Charest-Morin, R., Dea, N., Wilson, J.R., Dhall, S., Paquette, S., Street, J., Fisher, C.G., Dvorak, M.F., Shannon, C.P., Borchers, C., Balshaw, R., Foster, L.J., Kwon, B.K. &ldquo;Proteomic portraits reveal evolutionarily conserved and divergent responses to spinal cord injury.&rdquo; <em>bioRxiv</em>, doi: <a href="https://www.biorxiv.org/content/10.1101/2021.01.27.428528v1">10.1101/2021.01.27.428528</a>. </li>

    <li><u>Skinnider, M.A.</u><sup>†</sup>, Stacey, R.G., Wishart, D.S., Foster, L.J.<sup>†</sup> &ldquo;Deep generative models enable navigation in sparsely populated chemical space.&rdquo; <em>ChemRxiv</em>, doi: <a href="https://doi.org/10.26434/chemrxiv.13638347.v1">10.26434/chemrxiv.13638347.v1</a>. </li>

    <li><u>Skinnider, M.A.</u>*, Scott, N.E.*, Prudova, A., Stoynov, N., Stacey, R.G., Gsponer, J., Foster, L.J. &ldquo;An atlas of protein-protein interactions across mammalian tissues.&rdquo; <em>bioRxiv</em>, doi: <a href="https://www.biorxiv.org/content/10.1101/351247v1">10.1101/351247</a>. </li>

</ol>

**2021**

<ol class="publications" reversed start="29">

    <li>Squair, J.W.*<sup>†</sup>, <u>Skinnider, M.A.</u>*<sup>†</sup>, Gautier, M., Foster, L.J., Courtine, G.<sup>†</sup> &ldquo;Prioritization of cell types responsive to biological perturbations in single-cell data with Augur.&rdquo; <em>Nature Protocols</em>, to appear.</li>

    <li><u>Skinnider, M.A.</u>*<sup>†</sup>, Squair, J.W.*<sup>†</sup>, Kathe, C., Anderson, M.A., Gautier, M., Matson, K.J.E., Milano, M., Hutson, T.H., Barraud, Q., Phillips, A.A., Foster, L.J., La Manno, G., Levine, A.J., Courtine, G.<sup>†</sup> &ldquo;Cell type prioritization in single-cell data.&rdquo; <em>Nature Biotechnology</em>, <strong>39</strong>, 30&ndash;34 (2021). <a href="/files/Nat Biotechnol 2020 - Cell type prioritization in single-cell data.pdf">[pdf]</a> </li>

    <li><u>Skinnider, M.A.</u>, Cai, C., Stacey, R.G., Foster, L.J. &ldquo;PrInCE: an R/Bioconductor package for protein-protein interaction network inference from co-fractionation mass spectrometry data.&rdquo; <em>Bioinformatics</em>, doi: 10.1093/bioinformatics/btab022. <a href="/files/Bioinformatics 2021 - PrInCE- an R Bioconductor package for protein-protein interaction network inference from co-fractionation mass spectrometry data.pdf">[pdf]</a> </li>

    <li>Stacey, R.G., <u>Skinnider, M.A.</u>, Foster, L.J. &ldquo;On the robustness of graph-based clustering to random network alterations.&rdquo; <em>Molecular & Cellular Proteomics</em>, <strong>20</strong>, 100002 (2021). <a href="/files/MCP 2021 - On the Robustness of Graph-Based Clustering to Random Network Alterations.pdf">[pdf]</a> </li>

</ol>

**2020**

<ol class="publications" reversed start="25">

    <li><u>Skinnider, M.A.</u><sup>†</sup>, Johnston, C.W., Gunabalasingam, M., Merwin, N.J., Kieliszek, A.M., MacLellan, R.J., Li, H., Ranieri, M.R.M., Webster, A.L.H., Cao, M.P.T., Pfeifle, A., Spencer, N., To, Q.H., Wallace, D.P., Dejong, C.A.<sup>†</sup>, Magarvey, N.A. &ldquo;Comprehensive prediction of secondary metabolite structure and biological activity from microbial genome sequences.&rdquo; <em>Nature Communications</em>, <strong>11</strong>, 6058 (2020). <a href="/files/Nat Commun 2020 - Comprehensive prediction of secondary metabolite structure and biological activity from microbial genome sequences">[pdf]</a> </li>

    <li>Kerr, C.H.*, <u>Skinnider, M.A.</u>*, Andrews, D.D.T., Madero, A.M., Chan, Q.W.T., Stacey, R.G., Stoynov, N., Jan, E., Foster, L.J. &ldquo;Dynamic rewiring of the human interactome by interferon signalling.&rdquo; <em>Genome Biology</em>, <strong>21</strong>, 140 (2020). <a href="/files/Genome Biol 2020 - Dynamic rewiring of the human interactome by interferon signaling.pdf">[pdf]</a> </li>

    <li>Merwin, N.J.*, Mousa, W.K.*, Dejong, C.A., <u>Skinnider, M.A.</u>, Cannon, M.J., Li, H., Dial, K., Gunabalasingam, M., Johnston, C.W., Magarvey, N.A. &ldquo;DeepRiPP integrates multi-omics data to automate discovery of novel ribosomally synthesized natural products.&rdquo; <em>Proceedings of the National Academy of Sciences of the USA</em>, <strong>117</strong>, 371&ndash;380 (2020). <a href="/files/PNAS 2019 - DeepRiPP integrates multiomics data to automate discovery of novel ribosomally synthesized natural products.pdf">[pdf]</a> </li>

</ol>

**2019**

<ol class="publications" reversed start="22">

    <li><u>Skinnider, M.A.</u><sup>†</sup>, Squair, J.W., Foster, L.J.<sup>†</sup> &ldquo;Evaluating measures of association for single-cell transcriptomics.&rdquo; <em>Nature Methods</em>, <strong>16</strong>, 381&ndash;386 (2019). <a href="/files/Nat Meth 2019 - Evaluating measures of association for single-cell transcriptomics.pdf">[pdf]</a> </li>

</ol>

**2018**

<ol class="publications" reversed start="21">

    <li>Stacey, R.G., <u>Skinnider, M.A.</u>, Chik, J.H.L., Foster, L.J. &ldquo;Conditional interactions in literature-curated protein interaction databases.&rdquo; <em>BMC Genomics</em>, <strong>19</strong>, 758 (2018). <a href="/files/BMC Genomics 2018 - Context-specific interactions in literaturecurated protein interaction databases.pdf">[pdf]</a> </li>

    <li><u>Skinnider, M.A.</u>, Stacey, R.G., Foster, L.J. &ldquo;Genomic data integration systematically biases interactome mapping.&rdquo; <em>PLoS Computational Biology</em>, <strong>14</strong>, e1006474 (2018). <a href="/files/PLoS Comp Biol 2018 - Genomic data integration systematically biases interactome mapping.pdf">[pdf]</a> </li>

    <li>Squair, J.W.*, Tigchelaar, S., Moon, K.M., Liu, J., Tetzlaff, W., Kwon, B.K., Krassioukov, A.V., West, C.R., Foster, L.J., <u>Skinnider, M.A.</u>*<sup>†</sup>. &ldquo;Integrated systems analysis reveals conserved gene networks underlying response to spinal cord injury.&rdquo; <em>eLife</em>, <strong>7</strong>, e39188 (2018). <a href="/files/eLife 2018 - Integrated systems analysis reveals conserved gene networks underlying response to spinal cord injury.pdf">[pdf]</a> </li>

    <li><u>Skinnider, M.A.</u>, Twa, D.D.W., Squair, J.W., Rosenblum, N.D., Lukac, C.D. &ldquo;Predictors of sustained research involvement among Canadian MD/PhD program graduates.&rdquo; <em>Medical Education</em>, <strong>52</strong>, 536&ndash;545 (2018). <a href="/files/Med Educ 2018 - Predictors of sustained research involvement among MD/PhD programme graduates.pdf">[pdf]</a> </li>

    <li><u>Skinnider, M.A.</u>, Johnston, C.W., Merwin, N.J., Dejong, C..A., Magarvey, N.A. &ldquo;Global analysis of prokaryotic tRNA-derived cyclodipeptide biosynthesis.&rdquo; <em>BMC Genomics</em>, <strong>19</strong>, 45 (2018). <a href="/files/BMC Genomics 2018 - Global analysis of prokaryotic tRNA-derived cyclodipeptide biosynthesis.pdf">[pdf]</a> </li>

</ol>

**2017**

<ol class="publications" reversed start="16">

    <li>Stacey, R.G., <u>Skinnider, M.A.</u>, Scott, N.E., Foster, L.J. &ldquo;A rapid and accurate approach for prediction of interactomes from co-elution data (PrInCE).&rdquo; <em>BMC Bioinformatics</em>, <strong>18</strong>, 457 (2017). <a href="/files/BMC Bioinformatics 2017 - A rapid and accurate approach for prediction of interactomes from co-elution data (PrInCE).pdf">[pdf]</a> </li>

    <li>Twa, D.D.W., <u>Skinnider, M.A.</u>, Squair, J.W., Lukac, C.D. &ldquo;Cross-sectional-derived determinants of satisfaction with physician-scientist training among Canadian MD/PhD graduates. <em>PLoS ONE</em>, <strong>12</strong>, e0185218 (2017). <a href="/files/PLoS ONE 2017 - Cross-sectional-derived determinants of satisfaction with physician-scientist training among Canadian MD/PhD graduates.pdf">[pdf]</a> </li>

    <li><u>Skinnider, M.A.</u>, Dejong, C.A., Franczak, B.C., McNicholas, P.D., Magarvey, N.A. &ldquo;Comparative analysis of molecular similarity methods for modular natural products with a hypothetical structure enumeration algorithm.&rdquo; <em>Journal of Cheminformatics</em>, <strong>9</strong>, 46 (2017). <a href="/files/J Cheminf 2017 - Comparative analysis of chemical similarity methods for modular natural products with a hypothetical structure enumeration algorithm.pdf">[pdf]</a> </li>

    <li><small><em>(letter)</em></small> <u>Skinnider, M.A.</u>, Magarvey, N.A. &ldquo;Statistical reanalysis of natural products reveals increasing chemical diversity.&rdquo; <em>Proceedings of the National Academy of Sciences of the USA</em>, <strong>114</strong>, E6271&ndash;E6272 (2017). <a href="/files/PNAS 2017 - Statistical reanalysis of natural products reveals increasing chemical diversity.pdf">[pdf]</a> </li>

    <li><u>Skinnider, M.A.</u>, Merwin, N.J., Johnston, C.W., Magarvey, N.A. &ldquo;PRISM 3: An expanded resource for the automated prediction of microbial secondary metabolomes.&rdquo; <em>Nucleic Acids Research</em>, <strong>45</strong>, W49&ndash;W54 (2017). <a href="/files/NAR 2017 - PRISM 3- expanded prediction of natural product chemical structures from microbial genomes.pdf">[pdf]</a> </li>

    <li><u>Skinnider, M.A.</u>*, Squair, J.W.*, Twa, D.D.W.*, Ji, J.X.*, Kuzyk, A., Wang, X., Steadman, P.E., Zaslavsky, K., Dey, A.K., Eisenberg, M.J., Gagné, E.R., HayGlass, K.T., Lewis, J.F., Margetts, P.J., Underhill, D.A., Rosenblum, N.D., Raymond, L.A. &ldquo;Characteristics and outcomes of Canadian MD/PhD program graduates: a cross-sectional survey.&rdquo; <em>CMAJ Open</em>, <strong>5</strong>, E308&ndash;E314 (2017). <a href="/files/CMAJ Open 2017 - Characteristics and outcomes of Canadian MD/PhD program graduates- a cross-sectional survey.pdf">[pdf]</a> </li>

    <li>Streijger, F., <u>Skinnider, M.A.</u>, Rogalski, J.C., Balshaw, R., Shannon, C.P., Prudova, A., Belanger, L., Ritchie, L., Tsang, A., Christie, S., Parent, S., Thiong, J.M., Bailey, C., Urquhart, J., Ailon, T., Paquette, S., Boyd, M., Street, J., Fisher, C.G., Dvorak, M.F., Borchers, C.H., Foster, L.J., Kwon, B.K. &ldquo;Targeted proteomic analysis of human cerebrospinal fluid after acute spinal cord injury.&rdquo; <em>Journal of Neurotrauma</em>, <strong>34</strong>, 2054&ndash;2068 (2017). <a href="/files/J Neurotrauma 2017 - A Targeted Proteomics Analysis of Cerebrospinal Fluid after Acute Human Spinal Cord Injury.pdf">[pdf]</a> </li>

</ol>

**2016**

<ol class="publications" reversed start="9">

    <li><u>Skinnider, M.A.</u>*, Johnston, C.W.*, Edgar, R.E., Dejong, C.A., Merwin, N.J., Rees, P.N., Magarvey, N.A. &ldquo;Genomic charting of ribosomally synthesized natural product chemical space facilitates targeted mining.&rdquo; <em>Proceedings of the National Academy of Sciences of the USA</em>, <strong>113</strong>, E6343&ndash;E6351 (2016). <a href="/files/PNAS 2016 - Genomic charting of ribosomally synthesized natural product chemical space facilitates targeted mining.pdf">[pdf]</a> </li>

    <li>Dejong, C.A.*, Chen, G.M.*, Li, H.*, Johnston, C.W., Edwards, M., Rees, P.N., <u>Skinnider, M.A.</u>, Webster, A.L.H., Magarvey, N.A. &ldquo;Retrobiosynthesis of polyketide and nonribosomal peptide natural product chemistry and global alignment to biosynthetic clusters.&rdquo; <em>Nature Chemical Biology</em>, <strong>12</strong>, 1007&ndash;1014 (2016). <a href="/files/Nat Chem Biol 2016 - Polyketide and nonribosomal peptide retrobiosynthesis and global gene cluster matching.pdf">[pdf]</a> </li>

    <li>Johnston, C.W., <u>Skinnider, M.A.</u>, Dejong, C.A., Rees, P.N., Chen, G.M., Walker, C., French, S., Brown, E.D., Bérdy, J., Liu, D.Y., Magarvey, N.A. &ldquo;Assembly and clustering of natural antibiotics guides target identification.&rdquo; <em>Nature Chemical Biology</em>, 12, 233&ndash;239 (2016). <a href="/files/Nat Chem Biol 2016 - Assembly and clustering of natural antibiotics guides target identification.pdf">[pdf]</a> </li>

    <li>Johnston, C.W., Connaty, A.D., <u>Skinnider, M.A.</u>, Li, Y.C., Grunwald, A., Wyatt, M.A., Kerr, R.G., Magarvey, N.A. &ldquo;Informatic search strategies to discover analogues and variants of natural product archetypes.&rdquo; <em>Journal of Industrial Microbiology and Biotechnology</em>, <strong>43</strong>, 293&ndash;298 (2016). <a href="/files/JIMB 2015 - Informatic search strategies to discover analogues and variants of natural product archetypes.pdf">[pdf]</a> </li>

</ol>

**2015**

<ol class="publications" reversed>

    <li>Johnston, C.W.*, <u>Skinnider, M.A.</u>*, Wyatt, M.A., Li, X., Ranieri, M.R.M., Yang, L., Zechel, D.L., Ma, B., Magarvey, N.A. &ldquo;An automated Genomes-to-Natural Products platform (GNP) for the discovery of modular natural products.&rdquo; <em>Nature Communications</em>, <strong>6</strong>, 8421 (2015). <a href="/files/Nat Comms 2015 - An automated Genomes-to-Natural Products platform (GNP) for the discovery of modular natural products.pdf">[pdf]</a> </li>

    <li><small><em>(perspective)</em></small> Twa, D.D.W.*, Squair, J.W.*, <u>Skinnider, M.A.</u>*, Ji, J.X.*. &ldquo;The Canadian clinician-scientist training program must be reinstated.&rdquo; <em>Journal of Clinical Investigation</em>, <strong>125</strong>, 4317&ndash;4319 (2015). <a href="/files/JCI 2015 - The Canadian clinician-scientist training program must be reinstated.pdf">[pdf]</a> </li>

    <li><u>Skinnider, M.A.</u>, Dejong, C.A., Rees, P.N., Johnston, C.W., Li, H., Webster, A.L.H., Wyatt, M.A., Magarvey, N.A. &ldquo;Genomes to natural products prediction informatics for secondary metabolomes (PRISM).&rdquo; <em>Nucleic Acids Research</em>, <strong>43</strong>, 9645&ndash;9662 (2015). <a href="/files/NAR 2015 - Genomes to natural products PRediction Informatics for Secondary Metabolomes (PRISM).pdf">[pdf]</a> </li>

    <li>Yang, L.*, Ibrahim, A.*, Johnston, C.W.*, <u>Skinnider, M.A.</u>, Ma, B., Magarvey, N.A. &ldquo;Exploration of nonribosomal peptide families with an automated search algorithm.&rdquo; <em>Chemistry & Biology</em>, <strong>22</strong>, 1259&ndash;1269 (2015). <a href="/files/NAR 2015 - Genomes to natural products PRediction Informatics for Secondary Metabolomes (PRISM).pdf">[pdf]</a> </li>

    <li><u>Skinnider, M.A.</u>*, Johnston, C.W.*,  Zvanych, R., Magarvey, N.A. &ldquo;Automated identification of depsipeptide natural products by an informatic search algorithm.&rdquo; <em>ChemBioChem</em>, <strong>16</strong>, 223&ndash;227 (2015). <a href="/files/ChemBioChem 2015 - Automated Identification of Depsipeptide Natural Products by an Informatic Search Algorithm.pdf">[pdf]</a> </li>

</ol>

